StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -992,7 +992,7 @@ extension NativeFlowController: LinkLoginViewControllerDelegate {
         _ viewController: LinkLoginViewController,
         foundReturningUserWith lookupConsumerSessionResponse: LookupConsumerSessionResponse
     ) {
-        // TODO(BANKCON-11476): Save consumer publishable key for future use.
+        dataManager.consumerPublishableKey = lookupConsumerSessionResponse.publishableKey
```

Comment:
I don't know if this is necessary, but if you think there is some useful context of why we only set this here vs. other places, maybe worth adding

but, again, if there's no useful context then there's no need (ex. maybe we just set it here because this is one of the couple of places where it returns the key! then adding docs doesn't really add that much help)